### PR TITLE
New

### PR DESCRIPTION
**Pull request by mikhan333**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Information_Security_JS
-Client part of Encryption system for the Information Security course by Vladimirov at MIPT
+Cool Client part of Encryption system for the Information Security course by Vladimirov at MIPT
```
Comment (denisstasyev) [inline README.md]: I think it won't be cool 😎

Comment (mikhan333) [reply README.md]: Hm

Comment (mikhan333) [reply README.md]: Hard to say

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # Information_Security_JS
-Client part of Encryption system for the Information Security course by Vladimirov at MIPT
```
Comment (mikhan333) [inline README.md]: ...

Comment (denisstasyev) [reply README.md]: You need to commit this and try again

**Comment by mikhan333** [comment]
апвппаыв

**Pull request merged**